doc/modules/impute.rst

```diff
@@ -56,6 +57,23 @@ Note that, here, missing values are encoded by 0 and are thus implicitly stored
 in the matrix. This format is thus suitable when there are many more missing
 values than observed values.
 
+The :class:`SimpleImputer` class also supports categorical data represented as
+string values or pandas categoricals when using the ``most_frequent`` or
```

Comment:
nitpick: my preference would be to use 'most_frequent' rather than `most_frequent` for string values of parameters